CREDITS

```diff
@@ -0,0 +1,3 @@
+yt_astro_analysis is a group effort by the yt community.
+
+Thanks to everyone for all your contributions!
```

Comment:
only comment is that you might want to populate this file with the output of `git shortlog -nse`. You might also want to add the `.mailmap` file from yt to make `shortlog` output a deduplicated list of people.

Reply:
Ah, yeah, that's better :)